test/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/GoodsInVehicleControllerSpec.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.controllers
+
+import play.api.mvc.Result
+import play.api.test.Helpers._
+import uk.gov.hmrc.merchandiseinbaggagefrontend.model.core.YesNo
+import uk.gov.hmrc.merchandiseinbaggagefrontend.views.html.GoodsInVehicleView
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class GoodsInVehicleControllerSpec extends DeclarationJourneyControllerSpec {
+
+  private lazy val controller =
+    new GoodsInVehicleController(
+      controllerComponents, actionBuilder, declarationJourneyRepository, injector.instanceOf[GoodsInVehicleView])
+
+  private def ensureContent(result: Future[Result]) = {
+    val content = contentAsString(result)
```

Comment:
I guess this will be part of discussion, how much to test in the controller - in my view we shouldn't test contents but just the mechanics